src/audio/mic_privacy_manager/mic_privacy_manager_intel.c

```diff
@@ -0,0 +1,232 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2025 Intel Corporation. All rights reserved.
+
+#include "sof/audio/mic_privacy_manager.h"
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/mic_privacy/intel/mic_privacy.h>
+#include <zephyr/logging/log.h>
+#include <sof/audio/audio_stream.h>
+#include <sof/audio/buffer.h>
+#include <sof/audio/mic_privacy_manager.h>
+
+const struct device *mic_priv_dev;
+
+static struct mic_privacy_api_funcs *mic_privacy_api;
+static enum mic_privacy_policy mic_privacy_policy;
+
+#define LOG_DOMAIN mic_priv
+
+LOG_MODULE_REGISTER(LOG_DOMAIN);
+
+void handle_dmic_irq(void *const self, int a, int b)
+{
+	LOG_DBG("mic_privacy handle_dmic_irq");
```

Comment:
`LOG_*()` also include function names in the log entry, please consider removing in all such calls